### PR TITLE
[DOCS-160] Co-branded Pentest Reports

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
@yogipetkar Please review the updates describing co-branded pentest reports.

### CODE REVIEW

**Comment by mjang-cobalt** [comment]
@yogipetkar You can review how this will look in our docs at:

- https://deploy-preview-137--cobalt-docs.netlify.app/getting-started/checklist/#co-branded-reports
- https://deploy-preview-137--cobalt-docs.netlify.app/getting-started/glossary/#organization-owner (Select "Learn more" to see the new content)

**Review by mjang-cobalt — APPROVED** [review]
Thanks, @ana-dashuk-cobalt. Just a reminder: don't merge this PR until the engineering changes are merged on June 30.

**Comment by yogipetkar** [comment]
Hi @ana-dashuk-cobalt - Please take a look at following ZD article I have written. To enable partner attribute they need to reach out to Cobalt. Once such attribute is enabled they can managed to use the feature.
https://docs.google.com/document/d/1jS8FqJNYnpaoLXcc52XvRQrtnOT-mr7ITiip-D6spUA/edit?usp=sharing

File: content/en/Getting started/checklist.md [hunk 1]
```diff
@@ -51,6 +51,36 @@ Under the Asset tab, you can review:
...
+<li>Navigate to the <b>Settings</b> tab, and scroll down to the <b>Report Branding</b> section.</li>
+<li>Upload the logo image, and select <b>Apply</b>.</li></ul></li>
+
+<li>To verify that co-branding is enabled:
+<ul><li>Go to the <b>Pentests</b> tab, and select a pentest in the <i>Remediation</i> or <i>Closed</i> state.</li>
+<li>On the <b>Report</b> tab, download a report, and verify that it has your company logo.</li></ul></li>
+</ol>
+
+As a _Cobalt Partner_, you can edit your company logo or turn off co-branded reports for a client organization on the **Settings** tab under **Report Branding**.
+{{% /expand%}}
```
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/checklist.md:82]: @yogipetkar I've added more details about enabling co-branded reports. Please review.

File: content/en/Getting started/checklist.md [hunk 2]
```diff
@@ -51,6 +51,36 @@ Under the Asset tab, you can review:
...
+{{%expand "Learn more." %}}
+
+<ol>
+<li>To turn on the co-branding functionality, submit a request to <a href="mailto:support@cobalt.io">support@cobalt.io</a> with:
+<ul><li>Your name</li>
+<li>Your organization name</li>
+<li>List of client organizations you manage</li>
+<li>Reason: Enable partner co-branding functionality</li></ul></li>
+
+<li>Once we've processed your request, you become a <i>Cobalt Partner</i>, which you can verify on the <b>People</b> tab next to your name.</li>
```
Comment (yogipetkar) [inline content/en/Getting started/checklist.md:69]: Once we've processed your request, you have partner attributes, which ....

File: content/en/Getting started/checklist.md (hunk 1 above)
Comment (yogipetkar) [reply content/en/Getting started/checklist.md:82]: One minor comment added in line.

File: content/en/Getting started/checklist.md [hunk 3]
```diff
@@ -51,6 +51,36 @@ Under the Asset tab, you can review:
...
+{{%expand "Learn more." %}}
+
+<ol>
+<li>To turn on the co-branding functionality, submit a request to <a href="mailto:support@cobalt.io">support@cobalt.io</a> with:
+<ul><li>Your name</li>
+<li>Your organization name</li>
+<li>List of client organizations you manage</li>
+<li>Reason: Enable partner co-branding functionality</li></ul></li>
+
+<li>Once we've processed your request, you get access to specific functionality as a <i>Cobalt Partner</i>, which you can verify on the <b>People</b> tab next to your name.</li>
```
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/checklist.md:69]: @yogipetkar Updated. Please approve the PR if we are ready to go with this version.

**Pull request merged**